### PR TITLE
release prep for v4.5.8

### PR DESCRIPTION
**Pull request by ValClarkson**
[sc-15290]

**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

**What is the new behavior (if this is a feature change)?**

**Other information**:

### CODE REVIEW

File: docs/content/examples/postgresql/custom-config-ssl.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ sslkey=$CCPROOT/examples/kube/custom-config-ssl/certs/client.key"
 
 You should see a connection that looks like the following:
 ```
-psql (11.16)
+psql (11.17)
```
Comment (jmckulk) [inline docs/content/examples/postgresql/custom-config-ssl.md]: want to add the param here?

**Review by jmckulk — CHANGES_REQUESTED** [review]
I don't think we should be deleting the crunchy theme submodule. Other than that things are looking good

File: docs/content/examples/postgresql/custom-config-ssl.md (hunk 1 above)
Comment (ValClarkson) [reply docs/content/examples/postgresql/custom-config-ssl.md]: will do

**Comment by ValClarkson** [comment]
> I don't think we should be deleting the crunchy theme submodule. Other than that things are looking good

I didn't mean to delete it completely good catch i'll get that fixed

**Review by jmckulk — APPROVED** [review]

**Pull request merged**